scripts/db_migrator.py

```diff
@@ -715,11 +719,43 @@ def version_3_0_4(self):
         self.set_version('version_3_0_5')
         return 'version_3_0_5'
 
+
+
+
+
     def version_3_0_5(self):
         """
-        Current latest version. Nothing to do here.
+        Version 3_0_5
         """
         log.log_info('Handling version_3_0_5')
+        # Removing LOGLEVEL DB and moving it's content to CONFIG DB
+        # Removing Jinja2_cache
+        warmreboot_state = self.stateDB.get(self.stateDB.STATE_DB, 'WARM_RESTART_ENABLE_TABLE|system', 'enable')
+        if warmreboot_state == 'true':
+            table_name = "LOGGER"
+            loglevel_field = "LOGLEVEL"
+            logoutput_field = "LOGOUTPUT"
+            keys = self.loglevelDB.keys(self.loglevelDB.LOGLEVEL_DB, "*")
+            if keys is not None:
+                for key in keys:
+                    if key != "JINJA2_CACHE":
+                        fvs = self.loglevelDB.get_all(self.loglevelDB.LOGLEVEL_DB, key)
+                        component = key.split(":")[1]
+                        loglevel = fvs[loglevel_field]
+                        logoutput = fvs[logoutput_field]
+                        self.configDB.set(self.configDB.CONFIG_DB, '{}|{}'.format(table_name, component), loglevel_field, loglevel)
+                        self.configDB.set(self.configDB.CONFIG_DB, '{}|{}'.format(table_name, component), logoutput_field, logoutput)
+                        self.loglevelDB.delete(self.loglevelDB.LOGLEVEL_DB, key) ## todo: When we will remove the Jinja2 chace, this line need to get outside from the if scope outside the if scope.
```

Comment:
If you are depended on the J2 PR this should be marked on the PR for it to be merged after that PR and not have a separate PR to change this later on